fe/fe-core/src/main/java/com/starrocks/journal/bdbje/BDBEnvironment.java

```diff
@@ -578,8 +583,26 @@ public List<Long> getDatabaseNames() {
                 continue;
             }
 
-            long db = Long.parseLong(name);
-            ret.add(db);
+            if (prefix.isEmpty()) { // default GlobalStateMgr db
+                if (StringUtils.isNumeric(name)) {
```

Comment:
I suggest you add an assert somewhere to ensure that prefix is not a number. Otherwise we may mistakenly return all the dbs to global state manager.

Reply:
done